hoptimator-k8s/src/main/java/com/linkedin/hoptimator/k8s/K8sEngineTable.java

```diff
@@ -0,0 +1,91 @@
+package com.linkedin.hoptimator.k8s;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.Locale;
+import java.util.Optional;
+import java.util.stream.Collectors;
+import javax.sql.DataSource;
+
+import org.apache.calcite.adapter.jdbc.JdbcSchema;
+import org.apache.calcite.schema.Schema;
+
+import io.kubernetes.client.openapi.models.V1ObjectMeta;
+
+import com.linkedin.hoptimator.Engine;
+import com.linkedin.hoptimator.SqlDialect;
+import com.linkedin.hoptimator.k8s.models.V1alpha1Engine;
+import com.linkedin.hoptimator.k8s.models.V1alpha1EngineList;
+import com.linkedin.hoptimator.k8s.models.V1alpha1EngineSpec;
+import com.linkedin.hoptimator.util.HoptimatorJdbcSchema;
+
+
+public class K8sEngineTable extends K8sTable<V1alpha1Engine, V1alpha1EngineList, K8sEngineTable.Row> {
+
+  // CHECKSTYLE:OFF
+  public static class Row {
+    public String NAME;
+    public String URL;
+    public String DIALECT;
+    public String DRIVER;
+    public String[] DATABASES;
```

Comment:
nit: should prob be lowercase so we don't need to disable checkstyle

Reply:
This is a quirk of Calcite. These fields are referenced in the generated code, and things go sideways if they aren't all caps.